mailserver/mailserver.go

```diff
@@ -251,6 +254,15 @@ func (s *WMailServer) processRequest(peer *whisper.Peer, lower, upper uint32, bl
 	return ret
 }
 
+func (s *WMailServer) sendHistoricMessageAck(peer *whisper.Peer, request *whisper.Envelope) error {
+	hash := request.Hash()
+	envelope := &whisper.Envelope{
+		Data: hash[:],
```

Comment:
Passing only a request hash won't help with pagination or the client to figure out if the filters already been notified. I would add the hash of the last sent message to the payload as well. It should allow to resolve both issues.

Reply:
Or we could do it when working on pagination in the next PR. Up to you.

Reply:
ok maybe we can do it in the next PR so that `status-react` can start using it

Reply:
@adambabik I'm still working on timeouts and tests to push in this PR